youtube_dl/extractor/youku.py

```diff
@@ -100,15 +101,24 @@ def yk_t(s1, s2):
             del string_ls[idx]
 
         fileid_dict = {}
-        for format in data1['streamtypes']:
-            streamfileid = [
-                int(i) for i in data1['streamfileids'][format].strip('*').split('*')]
-            fileid = ''.join(
-                [shuffled_string_ls[i] for i in streamfileid])
-            fileid_dict[format] = fileid[:8] + '%s' + fileid[10:]
+        for stream in data1['stream']:
+            format = stream.get('stream_type')
+            #streamfileid = [
+            #    int(i) for i in data1['stream']['streamfileids']#[format].strip('*').split('*')]
+            #fileid = ''.join(
+            #    [shuffled_string_ls[i] for i in streamfileid])
```

Comment:
Thank you for the reminding. This is my first commit to a open source. I will try my best.